Wireframe/index.html

```diff
@@ -42,36 +44,36 @@ <h2>Branch</h2>
 
     <footer>
        <div class="footer-container">
-          <!-- Social Links Column -->
-          <div class="footer-column footer-social">
-            <h3>Follow Us</h3>
-            <a href="https://github.com/" target="_blank" aria-label="GitHub"><i class="fab fa-github"></i></a>
-            <a href="https://instagram.com/" target="_blank" aria-label="Instagram"><i class="fab fa-instagram"></i></a>
-            <a href="https://facebook.com/" target="_blank" aria-label="Facebook"><i class="fab fa-facebook-f"></i></a>
-            <a href="https://twitter.com/" target="_blank" aria-label="Twitter"><i class="fab fa-twitter"></i></a>
-          </div>
+            <!-- Social Links Column -->
```

Comment:
Instead of a div, what else could you use to signal a list of navigation links?

Reply:
Hi Sally,
Oh they could actually be replaced with <ul> and <li> elements. You are right. The HTML file has got more semantic style.
In order to do so, I’ve updated the code by replacing the `div` with a `<ul>` for the social links, as suggested. The links are now properly structured and styled. Thanks for pointing that out!

Let me know if it looks good.